src/test/java/Pages/HotelsHomepage.java

```diff
@@ -24,29 +34,70 @@
  public void clickSignInLink(){
      signInLocator.click();
  }
-
-    public void enterEmail (){
-     emailLocator.sendKeys("asdf@mail.com");
+ public void enterEmail (){
+     emailLocator.sendKeys("a^df@mail.com");
     }
-
     public void enterPassword(){
      passwordLocator.sendKeys("qwe1$29g");
     }
-
     public void clickSignInButton(){
      signInEmailPasswordLocator.click();
     }
-
     public boolean errorMsgDisplayed(){
      return signInErrorMsgLocator.isDisplayed();
     }
-
- public void clickSignUpLink(){
+    public void clickSignUpLink(){
      signUpLinkLocator.click();
  }
 public void clickOnTermsAndConditions(){ TermsConditionsLink.click();}
 public void clickOnCheckInCalendar(){checkInCalendar.click();}
 public void clickOnGuests(){guestLocator.click();}
+    public void enterIncorrectEmailAddress(){
+     emailLocator.sendKeys("#!@###");
+    }
+public void ifErrorEmailMessageDisplayed()
+{String expEmailMessage= "Please check your email address";
```

Comment:
Instead of hard coding here, you can make this method parameterized and pass the text from Feature file step.